jobpostings/views.py

```diff
@@ -30,4 +37,41 @@ def get(self, request):
                 } for job in job_group.job.all()],
             } for job_group in job_groups]
 
-        return JsonResponse({"message" : "SUCCESS", "result" : job_group_list}, status=200)
+        return JsonResponse({"message":"SUCCESS", "result" : job_group_list}, status=200)
+
+
+class PostingDetailView(View):
+    @lose_authorization
+    def get(self, request, posting_id):
+        user        = request.user
+        job_posting = JobPosting.objects.get(id=posting_id)
+        tags        = job_posting.tags.all()
+        bookmark    = Bookmark.objects.filter(user=user, job_posting=posting_id).exists()
+
+        job_posting_info={
+            "job_posting_id"    : job_posting.pk,
+            "job_posting_title" : job_posting.title,
+            "salary"            : job_posting.salary,
+            "due_date"          : job_posting.due_date,
+            "image_url"         : job_posting.image_url,
+            "job"               : job_posting.job.name,
+            "experience"        : job_posting.experience.name,
+            "bookmark"          : bookmark,
+            "company_info"      : {
+                "company_name" : job_posting.company.name,
+                "coordinate"   : job_posting.company.coordinate,
+            },
+            "description"       : {
+                "benefit"          : job_posting.benefit,
+                "intro"            : job_posting.description,
+                "main_task"        : job_posting.main_task,
+                "preference_point" : job_posting.preference,
+                "requirements"     : job_posting.requirement
+            },
+            "tag_info"          : [{
+                "name" : tag.name
+            } for tag in tags]
```

Comment:
```suggestion
            "tag_info"          : [{"name" : tag.name} for tag in tags]
```
그런데 이름만 줄꺼면 dictionary 말고 그냥 이름만 줘도 되지 않을까요?
```suggestion
            "tag_info"          : [tag.name for tag in tags]
```

Reply:
수정 했습니다.